modules/number-picker/main/index.coffee

```diff
@@ -6,6 +6,34 @@ checkValue = (numberPicker, context) ->
   if min isnt undefined then value = Math.max(value, min)
   if value isnt oldValue then context.value(value)
 
+addHoldHandler = (incrementOnHold, incrementDelay, selection, incrementFn) ->
+  if incrementOnHold
+    holdStart = undefined
+    holdTimeout = undefined
+    incrementInterval = undefined
+
+    clearTimers = ->
+      clearTimeout(holdTimeout)
+      clearInterval(incrementInterval)
+
+    selection.on 'pointerdown', 'hx.number-picker', (e) ->
+      holdStart = Date.now()
+      document.activeElement.blur()
```

Comment:
Forcing blur here makes it possible to type in the input and then increment/decrement without having to click outside of the number picker first.